msteams-platform/m365-apps/combine-office-add-in-and-teams-app.md

```diff
@@ -0,0 +1,626 @@
+
+    ```
+    {
+        "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+        "contentVersion": "1.0.0.0",
+        "parameters": {
+          "resourceBaseName": {
+            "value": "tabandaddin${{RESOURCE_SUFFIX}}"
+          },
+          "webAppSku": {
+            "value": "F1"
+          },
+          "storageSku": {
+            "value": "Standard_LRS"
+          }
+        }
+    }
+    ```
+
+1. Open the infra/azure.bicep file in the root of the project (not the one in either the tab or add-in subfolders) and replace its contents with the following code.
+
+    ```
+    // Params for Teams tab resources
+    @maxLength(20)
+    @minLength(4)
+    @description('Used to generate names for all resources in this file')
+    param resourceBaseName string
+    param webAppSku string
+    param serverfarmsName string = resourceBaseName
+    param webAppName string = resourceBaseName
+    param location string = resourceGroup().location
+    param storageSku string
+    param storageName string = resourceBaseName
+
+    module tabModule '../tab/infra/azure.bicep' = {
+      name: 'tabModule'
+      params: {
+        resourceBaseName: resourceBaseName
+        webAppSku: webAppSku
+        serverfarmsName: serverfarmsName
+        webAppName: webAppName
+        location: location
+      }
+    }
+
+    module addinModule '../add-in/infra/azure.bicep' = {
+      name: 'addinModule'
+      params: {
+        resourceBaseName: resourceBaseName
+        storageSku: storageSku
+        storageName: storageName
+        location: location
+      }
+    }
+    
+    // The output will be persisted in .env.{envName}. Visit https://aka.ms/teamsfx-actions/arm-deploy for more details.
+    output TAB_AZURE_APP_SERVICE_RESOURCE_ID string = tabModule.outputs.TAB_AZURE_APP_SERVICE_RESOURCE_ID // used in deploy stage
+    output TAB_DOMAIN string = tabModule.outputs.TAB_DOMAIN
+    output TAB_ENDPOINT string = tabModule.outputs.TAB_ENDPOINT
+    
+    output ADDIN_AZURE_STORAGE_RESOURCE_ID string = addinModule.outputs.ADDIN_AZURE_STORAGE_RESOURCE_ID // used in deploy stage
+    output ADDIN_DOMAIN string = addinModule.outputs.ADDIN_DOMAIN
+    output ADDIN_ENDPOINT string = addinModule.outputs.ADDIN_ENDPOINT
+    ```
+
+1. In Visual Studio Code open the Teams Toolkit and in the **ACCOUNTS** section be sure you're signed into your *Azure* account (in addition to being signed into your Microsoft 365 account). For more information about signing in, open [Exercise - Create Azure resources to host a Teams tab app](/training/modules/teams-toolkit-vsc-deploy-apps/03-create-azure-resources-exercise) and scroll to the **Sign in to Azure in Teams Toolkit** section.
+1. In the **LIFECYCLE** section of Teams Toolkit, select **Provision**. It can take several minutes. You may be prompted to select one of your Azure resource groups.
```

Comment:
```suggestion
1. In the **LIFECYCLE** section of Teams Toolkit, select **Provision**. 
    It can take several minutes. You might be prompted to select one of your Azure resource groups.
```